nucliadb_vectors/src/data_point_provider/mod.rs

```diff
@@ -48,8 +48,8 @@ pub enum VectorErr {
     IoErr(#[from] io::Error),
     #[error("Error in data point: {0}")]
     Dp(#[from] DPError),
-    #[error("Error in disk: {0}")]
-    Disk(#[from] DiskErr),
+    #[error("Error in fs: {0}")]
+    FsError(#[from] FsError),
```

Comment:
Ditto

Reply:
Same response as with the `FsError` suggestion.